bundler/default/lib/saito/peer.js

```diff
@@ -113,6 +113,25 @@ class Peer {
         return 0;
     }
 
+
+  //
+  // returns true if we are the first listed peer in the options file
+  // TODO -- isFirstPeer
+  isMainPeer() {
+    if (this.app.options) {
+      if (this.app.options.peers) {
+        if (this.app.options.peers.length > 0) {
```

Comment:
Instead of nested `if`s [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) will work:
`if (this.app?.options?.peers?.length > 0) {`

Reply:
Dear God, yes.

implementing codebase-wise has just hit my "stuff pending refactor" list

Reply:
If you just do it while you're at it, you can skip the "big refactor". Also... "big refactor" never happens.